README.md

```diff
@@ -39,15 +39,25 @@ Thus, our data layer has to rely only on the data emitted by the EVM. It cannot
 
 More components, e.g. a watchtower to challenge invalid state transitions, will be added in the future.
 
-## Installation
+## Getting Started
+
+### Requirements
+
+- Rust ^1.37
+- NodeJS ^11.0, starting with version 12 some deprecated APIs were removed that cause `scrypt`, `keccak`, `secp256k1`, and `sha3` packages to fail to build
```

Comment:
Doesn't the superscript on `^11.0` include `12` as acceptable version? Or does the `^` only allow all `11.X.X` versions?

Reply:
`^x.y.z <=> >=x.y.z && <(x+1).0.0`